scripts/SortHelper.js

```diff
@@ -0,0 +1,29 @@
+var _ = require('underscore');
+
+var SortMethods = {
+  sortDate: function(data,sortColumn,sortAscending) {
+
+      var sortedData = data.sort(function(a,b) {
```

Comment:
Maybe use similar naming to [Lodash](https://github.com/lodash/lodash/blob/master/lodash.js#L435). That way you could have something like 

```
data.sort(function(object, other) {
...
objectDateDefined
otherDateDefined
```